home-assistant-plugin/custom_components/xknx/binary_sensor.py

```diff
@@ -27,10 +30,19 @@ def __init__(self, device: XknxBinarySensor):
     def async_register_callbacks(self):
         """Register callbacks to update hass after device was changed."""
 
-        async def after_update_callback(device):
+        async def after_update_callback(device: XknxBinarySensor):
             """Call after device was updated."""
             self.async_write_ha_state()
 
+            self.hass.bus.fire(
+                f"knx_{self.entity_id}",
```

Comment:
```suggestion
                f"{DOMAIN}_{self.entity_id}",
```
We could avoid constants in code if we import the DOMAIN constant. The custom component would then fire a `xknx_binary_sensor` event instead of a `knx_binary_sensor` event - don't know if this makes sense.
I'm good with either option.